CorsixTH/Lua/hospital.lua

```diff
@@ -612,6 +620,10 @@ function Hospital:afterLoad(old, new)
     end
   end
 
+  if old < 154 then
+    self.overdraft_interest_rate = self.interest_rate + 0.02
```

Comment:
Could the overdraft differential be attempted to be got from the config instead? Though I'd also be happy with this if it's to play safe.

Reply:
They don't exist yet. It could re read the level file, but I'd rather the less surprising thing of they continue with what they started.